internal/transport/transport_test.go

```diff
@@ -427,8 +428,16 @@ func setUpServerOnly(t *testing.T, port int, sc *ServerConfig, ht hType) *server
 	return server
 }
 
-func setUp(t *testing.T, port int, ht hType) (*server, *http2Client, func()) {
-	return setUpWithOptions(t, port, &ServerConfig{}, ht, ConnectOptions{})
+// isGreetingDone verifies that client-server setup is complete
+// for the test.
+var isGreetingDone = atomic.Bool{}
+
+func setUp(t *testing.T, port int, ht hType, options ...ConnectOptions) (*server, *http2Client, func()) {
+	var copts = ConnectOptions{}
+	if len(options) > 0 {
+		copts = options[0]
+	}
+	return setUpWithOptions(t, port, &ServerConfig{}, ht, copts)
 }
```

Comment:
What is this all about?  You added an options parameter to `setUp`, but we already have `setUpWithOptions`.  I don't get it.

Reply:
Updated